Database/Corrections/sodQuestFixes.lua

```diff
@@ -245,6 +245,11 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.finishedBy] = {nil,{410168}},
             [questKeys.requiredRaces] = raceIDs.UNDEAD,
         },
+        [78229] = {
+            [questKeys.preQuestGroup] = {90088,90089,90090},
```

Comment:
This is a nice idea, but doesn't make any difference since the fake quests are never in the quest log or marked as "complete".

We can leave it in though, as we might add logic for this at some point 🤷🏻

Reply:
I'll leave it in there for the moment in the case that we will need it later. Can remove it in the next sub-major release if it still is not used.